Templates/Ros2FleetRobotTemplate/README.md

```diff
@@ -15,30 +15,109 @@ If your simulation does not match this and you would like to start with a simple
 Please follow the instructions in [ROS 2 Gem documentation](https://development--o3deorg.netlify.app/docs/user-guide/interactivity/robotics/project-configuration/)
 to install all required dependencies and create your project with a template (make sure to use chose this template during the process).
 
-## Spawning robots
+## Fleet navigation
 
-The level contains spawn points configured to easily add more Proteus robots through ROS 2 calls.
```

Comment:
How about to leave this part as "Simple spawning" and structure in a way that users know to follow either simple spawning or running the fleet example?

Reply:
I've put the "simple spawning" section back, with a minor notice, that for the fleet nav, the scene must be empty (robots are spawning automatically).